src/TraceEvent/Computers/TraceProcess.cs

```diff
@@ -99,8 +99,8 @@ public static void SetupCallbacks(TraceEventDispatcher source)
 
             kernelParser.ProcessEndGroup += delegate (ProcessTraceData data)
             {
-                data.Process().ProcessEnd(data);
-                // Don't filter them out (not that many, useful for finding command line)
+                data.source.Processes().TerminateProcess(data);
```

Comment:
I would prefer a slight reorganization/renaming.    I think 'ProcessEnd() should just do everything (you don't need to make up a new concept at this level of abstraction)    However you should remove ProcessEnd() from 'TerminateProcess() and call the method 'CleanupOldProcess() and call it from EndProcess().    From what I can tell this is equivalent  and is clearer (we are not introducing a new concept, we are simply having the existing EndProcess do some more work (cleaning up old processes).

Reply:
Done.